go.mod

```diff
@@ -36,9 +38,9 @@ require (
 	github.com/prometheus/client_golang v1.19.1
 	github.com/prometheus/client_model v0.6.1
 	github.com/prometheus/common v0.55.0
-	github.com/rancher/fleet/pkg/apis v0.10.0-rc.16
-	github.com/rancher/lasso v0.0.0-20240424194130-d87ec407d941
-	github.com/rancher/wrangler/v2 v2.1.4
+	github.com/rancher/fleet/pkg/apis v0.10.0-rc.18
+	github.com/rancher/lasso v0.0.0-20240430201833-6f3def65ffc5
+	github.com/rancher/wrangler/v3 v3.0.0-rc2
```

Comment:
Do we want to use an RC of Wrangler in a new Fleet release (0.10.0)?
On the other hand, `rancher/rancher` does it, yes...

Reply:
> Do we want to use an RC of Wrangler in a new Fleet release (0.10.0)? On the other hand, `rancher/rancher` does it, yes...

Looks strange, right?